minicluster/src/main/java/org/apache/accumulo/minicluster/MiniAccumuloRunner.java

```diff
@@ -75,6 +75,7 @@
   private static final String NUM_T_SERVERS_PROP = "numTServers";
   private static final String DIRECTORY_PROP = "directory";
   private static final String INSTANCE_NAME_PROP = "instanceName";
+  private static final String EXISTING_ZOO_KEEPERS_PROP = "existingZooKeepers";
```

Comment:
Did we already address the addition here? MiniAccumuloRunner is also public API (like MiniAccumuloCluster). Seems odd to add it here. I don't think we have a good way to hide it though (like the MiniAccumuloClusterImpl), so maybe it's ok?